ecs-init/docker/docker.go

```diff
@@ -127,6 +127,9 @@ const (
 	execAgentLogRelativePath = "/exec"
 )
 
+var execCommand = exec.Command
```

Comment:
nit: can you have all vars into one block?

Reply:
moved to a common var